### PR TITLE
Maven task

### PR DESCRIPTION
**Pull request by Shake-Ghairbyan**
I had hard times with configuration to execute main, but eventually, it worked.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea/
+target/
```
Comment (chandranik90) [inline .gitignore:4]: Why .iml file has been removed  from gitignore?

File: src/main/java/helloWorld/Main.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package helloWorld;
```
Comment (chandranik90) [inline src/main/java/helloWorld/Main.java]: The package name should be all lowercase

**Review by chandranik90 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Shake-Ghairbyan) [reply .gitignore:4]: it was not generated with maven. If we continue with maven, should I still keep it?

Comment (Shake-Ghairbyan) [reply .gitignore:4]: Apparently, I just realized I did not pull the latest `main` to my local `main`, so I branched out `maven_task` from the outdated version of local `main`. Hence, the .gitignore file was generated from scratch and was missing `*.iml` entry. 

I just merged the latest main into this branch, so it should be ready now. 

Please, re-review.

**Review by chandranik90 — APPROVED** [review]

**Pull request merged**